src/Monolog/Logger.php

```diff
@@ -614,4 +614,14 @@ public function emergency($message, array $context = array())
     {
         return $this->addRecord(static::EMERGENCY, $message, $context);
     }
+
+    /**
+     * Set the timezone to be used for the timestamp of log records.
+     *
+     * @param string $tz Timezone name
+     */
+    public static function setTimezone($tz)
```

Comment:
Since ultimately we end up with a DateTimeZone object I'd say yes accepting that makes it easier to handle validation and so on upstream if needed, and people might already be working with a DTZ object in their app anyway. I'll do the change after merge though.